vaadin-upload-flow-parent/vaadin-upload-flow-integration-tests/src/test/java/com/vaadin/flow/component/upload/tests/UploadI18nIT.java

```diff
@@ -69,6 +69,7 @@ public void testFullI18nShouldOverrideCompleteConfigurationInWebComponentPropert
 
         UploadI18N expected = UploadTestsI18N.RUSSIAN_FULL;
         JsonObject expectedJson = (JsonObject) JsonSerializer.toJson(expected);
+        deeplyRemoveNullValuesFromJsonObject(expectedJson);
```

Comment:
These two tests expect keys for null values to be present in the actual map. However, they are removed from the map when setting the i18n.